src/pages/components/tile/usage.mdx

```diff
@@ -366,88 +401,95 @@ the chevron icon.
 <Row>
 <Column colLg={8}>
 
-![Tile anatomy](images/tile-usage-6.png)
+![Tile anatomy](images/tile-usage-8.png)
 
 </Column>
 </Row>
 
-1. **Container**: Includes a title, optional label, and the close icon.
+1. **Image (optional)** : The image follows the aspect ratios with the default
+   size of 4:3
+
+2. **Container**: Contains the content area
 
-2. **Content area**: Contains the information and/or controls needed to complete
+3. **Content area**: Contains the information and/or controls needed to complete
    the modal's task. It can include message text and components.
 
 ### Sizing
 
 The **width** varies depending on three basic grid modes: wide, narrow, and
```

Comment:
Tile width varies depending on three basic grid modes: wide, narrow, and condensed. Tile height varies depending on the amount of content placed within it, but applies the [spacing scale](https://carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale) and follows Carbon’s standard [aspect ratio](https://www.carbondesignsystem.com/guidelines/2x-grid/overview/#aspect-ratio)s. The minimum height starts with 2:1 aspect ratio. As the content grows, apply appropriate vertical spacers between sections within the content area.